em_workflows/file_path.py

```diff
@@ -248,8 +248,11 @@ def copy_workdir_logs_to_assets(self) -> Path:
             f"{self.assets_dir.as_posix()}/{dir_name_as_date}/{self.fp_in.stem}"
         )
         if dest.exists():
-            log(f"Output assets log directory already exists! removing: {dest}")
-            shutil.rmtree(dest)
+            log(f"Output already exists! removing: {dest}")
+            if dest.is_dir():
+                shutil.rmtree(dest)
+            else:
+                dest.unlink()
```

Comment:
Is there a chance `dest` is going to be a file here? Rest of the code assumes dest is a directory.